src/utilities/omit.js

```diff
@@ -0,0 +1,5 @@
+export default function omit(object = {}, ...keys) {
+  const copy = Object.assign({}, object);
+  keys.forEach(key => delete copy[key]);
+  return copy;
+}
```

Comment:
> That would the time complexity O(objSize * keysLength) instead of O(objSize + keysLength)

you are right, but your proposal is not possible to optimise. Mine will be warmed and jit-ed. So my proposal will be slower only first time, i would say